src/App/Components/Common/NavProfile/navbar.js

```diff
@@ -17,24 +42,22 @@ export class navbar extends Component {
           aria-controls="navbarNavAltMarkup"
           aria-expanded="false"
           aria-label="Toggle navigation"
+          onMouseOver={this.hideNotificationsComponent}
         >
-          <span className="navbar-toggler-icon" />
+          <i className="fas fa-bars" />
         </button>
         <div className="collapse navbar-collapse nav-links-container" id="navbarNavAltMarkup">
-          <div className="navbar-nav">
-            <Link className="nav-item nav-link" to="/login">
-              Login
-            </Link>
-            <div className="nav-item nav-link to-hide-on-responsive">|</div>
-            <Link className="nav-item nav-link" to="/signup">
-              Signup
-            </Link>
-          </div>
+          {!localStorage.getItem('token') ? <NavLinks /> : <Profile />}
```

Comment:
@nkpremices why access the localStorage here directly?

Reply:
I was trying to switch those two components there. Should I put it in a variable ? @MemunaHaruna

Reply:
@nkpremices see my comment below on localStorage usage